pkgs/applications/audio/cmusfm/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, curl, libnotify, gdk_pixbuf }:
+
+stdenv.mkDerivation rec {
+  version = "ad2fd0aad3f4f1a25add1b8c2f179e8859885873";
+  name = "cmusfm-${version}";
+  src = fetchFromGitHub {
+    owner = "Arkq";
+    repo = "cmusfm";
+    rev = "${version}";
+    sha256 = "0wpwdwgyrp64nvwc6shy0n387p31j6aw6cnmfi9x2y1jhl5hbv6b";
+  };
+  # building
+  configureFlags = "--enable-libnotify";
+  autoreconfPhase = "autoreconf --install";
+  nativeBuildInputs = [ autoreconfHook pkgconfig curl libnotify gdk_pixbuf ];
```

Comment:
The program executed during build should go to `nativeBuildInputs`, the libraries used at run-time should go to `buildInputs`.